data-raw/dm_metabolic.R

```diff
@@ -0,0 +1,46 @@
+# Datasets: dm_metabolic
```

Comment:
```suggestion
# Dataset: dm_metabolic
```

Reply:
Nice - thank you! Grammar is fixed and commited :)